### ISSUE TITLE
Backup fails on root subvolume labelled @ on Ubuntu

### ISSUE THREAD

**Issue by timharton** [digint/btrbk#3]
File root subvolume on my file system is labelled `@`  following the ubuntu naming convention (https://help.ubuntu.com/community/btrfs). 
When I perform a dryrun using this label I get the error:
```
btrbk command line client, version 0.12  (Thu Mar 19 09:16:40 2015)
ERROR: Ambiguous file for option "subvolume" in "btrbk-#####.conf" line 83: @
ERROR: Failed to parse configuration file
```

This also happens for my subvolume labelled `@home`.

```
btrbk command line client, version 0.12  (Thu Mar 19 09:17:40 2015)
ERROR: Ambiguous file for option "subvolume" in "btrbk-#####.conf" line 88: @home
ERROR: Failed to parse configuration file
```

**Comment by digint** [comment]
btrbk is very strict when it comes to input validation (for security reasons). However, the '@' character should not impose any problems here.

Changed file match from `/[0-9a-zA-Z_\-\.\/]+/` to `/[0-9a-zA-Z_@\-\.\/]+/` in commit:  e39263588d576692a3adcd10bfa2ee6f6b146574, this should satisfy all ubutu people out there (except for people using other weird characters for their subvolume naming)

**Comment by digint** [comment]
Fixed in btrbk v0.13

Created new release as this bug was a blocker for the ubuntu community.
Thanks @timharton for pointing me to this.
